exercicios_projeto/Client/Client.js

```diff
@@ -1,36 +1,45 @@
 
 class Client {
-    id;
     name;
     #cpf;
     email;
     phone;
     #income;
-    
-    
-    //static createdClients = [];
+        
+    static createdClients = [];
 
     constructor(name, cpf, income) {
         this.name = name;
         this.#cpf = cpf;
         this.income = income;
         
+        
+        // this.verifyCpf();
+
+        Client.createdClients.push(this)
     }
 
     get cpf() {
-        //to do
+        return this.#cpf;
+    }
+    
+    get income() {
+        return this.#income;
     }
 
-    createClient() {
-        //to do
+    set income(income) {
+        this.#income = income;
     }
 
-    addEmail() {
-        //to do
+    addEmail(email) {
+        this.email = email;
+        Client.createdClients.email = this.email;
```

Comment:
Se o objetivo do seu código na linha 36 for adicionar o email cadastrado ao array createdClients[ ] , acredito que o seu código da linha 35 já faz isso, ao atribuir valor this.email ( que inicia undefined).